function-service-impl/src/main/java/com/future/function/service/impl/feature/communication/questionnaire/MyQuestionnaireServiceImpl.java

```diff
@@ -140,47 +132,126 @@ public QuestionnaireParticipant getQuestionnaireParticipantById(
   }
 
   @Override
-  public QuestionnaireResponse createQuestionnaireResponseToAppraiseeFromMemberLoginAsAppraiser(
+  public void createQuestionnaireResponseToAppraiseeFromMemberLoginAsAppraiser(
+    Questionnaire questionnaire,
+    List<QuestionResponseQueue> questionResponses,
+    User memberLogin,
+    User appraisee
+  ) {
+    questionResponseQueueRepository.save(questionResponses);
+
+    if (!questionnaireResponseRepository
+          .findByQuestionnaireAndAppraiseeAndAppraiserAndDeletedFalse(
+              questionnaire, appraisee, memberLogin)
+      .isPresent()) {
+      Answer answer = Answer.builder()
+        .maximum(0F)
+        .minimum(0F)
+        .average(0F)
+        .build();
+
+      QuestionnaireResponse questionnaireResponse =
+        QuestionnaireResponse.builder()
+          .questionnaire(questionnaire)
+          .appraisee(appraisee)
+          .appraiser(memberLogin)
+          .scoreSummary(answer)
+          .build();
+      this.questionnaireResponseRepository.save(questionnaireResponse);
+    }
+  }
+
+  public void updateUserSummary(
     Questionnaire questionnaire, List<QuestionResponse> questionResponses,
     User memberLogin, User appraisee
   ) {
-
     Answer scoreSummary = Answer.builder()
       .maximum(0F)
       .minimum(6F)
       .build();
 
     Float avarageScore = new Float(0.0);
 
+    System.out.println(questionResponses.size());
```

Comment:
is this necessary? if so, please use `@Slf4j` and `log.info` instead of `println`